src/components/layout/FixedLeft/FixedLeft.jsx

```diff
@@ -1,14 +1,25 @@
-import _ from 'lodash';
-import $ from 'jquery';
+import { isNull, clamp } from 'lodash';
 import React from 'react';
 import PropTypes from 'prop-types';
-import Base from '../../Base.jsx';
 import style from './style.scss';
+import getDataAttrs from '../../../getDataAttrs';
 
 /**
   * The fixed left page is great.
  */
 
+/* eslint-disable */
+const getOffsetTop = (elem) => {
+  let offsetTop = 0;
+  do {
```

Comment:
whoa. never seen a `do while` in js.

perhaps more idiomatic to use `while (...) {...}`